Add a how to enable a bootchimeDXE

ZISQO opened the pull request:
for all of opencore bootloader users.

ZISQO committed:
Update README.md

Add a how to enable a bootchimeDXE

ZISQO committed:
Update README.md

Add a how to enable a bootchimeDXE

ZISQO committed:
Update README.md

Add a how to enable a bootchimeDXE

Goldfish64 commented:
I don't believe its necessary to have such instructions front and center, as people using OC should have an understanding of adding EFI drivers already, with some of the info in that section already explained elsewhere. I still may merge it in, but please relocate the section towards the bottom of the README, and recheck for spelling.

Thanks.

ZISQO committed:
Update README.md

어르마나 잘났기르래 부후팅쇼리가 늦케 나호는거또 몬꼬치고 지틀 잘난맛헤 살코있냐
븅쉰들히 아쥬 지둘먄 쟐햐는쥴알햐 단디 꼼꼼햐게 셜명셔를 친졀하궤 만틀라교 닝겐~
